contracts/BC_fusion/BSCGovernor.sol

```diff
@@ -41,12 +41,30 @@ contract BSCGovernor is
     /*----------------- errors -----------------*/
     error NotWhitelisted();
     error TotalSupplyNotEnough();
+    error GovernorPaused();
+    error OnlyGovernorProtector();
+
+    /*----------------- events -----------------*/
+    event Paused();
+    event Resumed();
 
     /*----------------- storage -----------------*/
     // target contract => is whitelisted for governance
     mapping(address => bool) public whitelistTargets;
-
     bool public proposeStarted;
+    bool private _paused;
```

Comment:
why private? I would like to know whether this contract is paused or not now.

Reply:
modified to public